torch/_export/converter.py

```diff
@@ -676,7 +679,15 @@ def convert_node(self, node: torch._C.Node):
         # matching converter for that.
         handler_func_name = ir_name_to_func_name(node_kind)
         handler_func = getattr(self, handler_func_name, self.convert_call_function_op)
-        handler_func(node)
+        
+        log_msg = f"Convert [{str(node)[:-1]}]\nConvert using [{handler_func.__name__}] "
+        try:
+            handler_func(node)
+        except Exception as e:
+            log_msg += "fails with {e}"
+            raise
+        log_msg += "succeeds"
+        log.debug(log_msg)
```

Comment:
I'm not sure if we need a "succeeds" logging, since I would assume if we move onto the next node, it would imply we did the previous one successfully.